src/mame/heathkit/tlb.cpp

```diff
@@ -262,6 +278,34 @@ void heath_tlb_device::mm5740_data_ready_w(int state)
 	}
 }
 
+void heath_tlb_device::check_for_reset()
+{
+	if (m_reset_key && m_right_shift)
+	{
+		m_reset_pending = true;
+	}
+	else if (m_reset_pending)
+	{
+		m_reset_pending = false;
+		reset();
+		m_reset(ASSERT_LINE);
```

Comment:
Should this be immediately followed by an `m_reset(CLEAR_LINE);`? I've seen that for some other systems.

Reply:
I looks like it is actually held in reset while the keys are both pressed. When one of the keys is released, the reset line is cleared.